src/openrct2/entity/Peep.cpp

```diff
@@ -383,7 +383,7 @@ void Peep::StateReset()
 }
 
 /** rct2: 0x00981D7C, 0x00981D7E */
-static constexpr CoordsXY word_981D7C[4] = {
+static constexpr CoordsXY walkingOffsetByDirection[4] = {
```

Comment:
```suggestion
static constexpr CoordsXY walkingOffsetByDirection[kNumOrthogonalDirections] = {
```
It’s possible that this constant has not been renamed yet. (In which case it will still be called kNumOrthogonalDirections`.

Reply:
I see we have a `NumOrthogonalDirections` in Location.hpp, indeed. I'll use and rename the constant in a separate commit.